### PR TITLE
BiDi: consolidate BiDi fixtures in webdriver/tests/support/fixtures_bidi

### PR DESCRIPTION
**Pull request by thiagowfx**
This fixtures file is already globally included in webdriver/tests/conftest.py via `pytest_plugins`.

Stop having webdriver/tests/bidi/conftest.py be a duplicate of the support fixture.

Originally introduced via 228559b5644d05a3ec4aba4463adc163fc307065

### CODE REVIEW

**Comment by sadym-chromium** [comment]
@whimboo do you want to take a look?

**Comment by whimboo** [comment]
@sadym-chromium I actually wonder if we should move the bidi fixtures from `/support` to `bidi/` and have them all in the `conftest.py`. None of them are interesting for classic and only those affecting both protocols might be suited well at this level. What do you think?

File: webdriver/tests/support/fixtures_bidi.py
Comment (sadym-chromium) [reply webdriver/tests/support/fixtures_bidi.py:20]: sorry, forgot to publish this comment yesterday

**Comment by sadym-chromium** [comment]
> @sadym-chromium I actually wonder if we should move the bidi fixtures from `/support` to `bidi/` and have them all in the `conftest.py`. None of them are interesting for classic and only those affecting both protocols might be suited well at this level. What do you think?

I think it makes sense.

**Comment by whimboo** [comment]
> > @sadym-chromium I actually wonder if we should move the bidi fixtures from `/support` to `bidi/` and have them all in the `conftest.py`. None of them are interesting for classic and only those affecting both protocols might be suited well at this level. What do you think?
> 
> I think it makes sense.

Actually lets wait with that. I'll have a topic for the next monthly meeting about moving all HTTP tests into a `webdriver/tests/http/` subfolder. So such a refactoring might be better done with such a change.

For now I would be fine in moving them all into the bidi fixture file.

**Review by whimboo — CHANGES_REQUESTED** [review]
As what @sadym-chromium said. Lets revert the formatting changes.

File: webdriver/tests/support/fixtures_bidi.py
Comment (thiagowfx) [reply webdriver/tests/support/fixtures_bidi.py:20]: Ooops, that's auto-formatting from VSCode. Will revert those changes.

**Comment by thiagowfx** [comment]
@whimboo Done!

(Apologizes for the delay, my notification settings were disabled for this repo, they are now enabled.)

**Review by sadym-chromium — APPROVED** [review]

**Comment by thiagowfx** [comment]
@whimboo over to you

**Comment by whimboo** [comment]
None of these fixtures that we are going to move here are really directly related to Webdriver BiDi only. They could be used for tests of both protocols. Note that I've also moved the [`get_test_page` fixture](https://github.com/web-platform-tests/wpt/blob/ea2a1693d33f702d1557e322cc32b6b3e9162d0e/webdriver/tests/support/fixtures.py#L255) to the top-level `fixtures.py` file.

I wonder if we should at least move the above fixtures to that file as well - close to the `get_test_page` fixture. Eventually we most likely want to get rid of those and only have `get_test_page` but I don't want to stress on that for this particular PR.

What do you think? 

(Note that there are merge conflicts now)

**Comment by thiagowfx** [comment]
@whimboo Done.

**Review by whimboo — APPROVED** [review]

**Comment by whimboo** [comment]
@jgraham could you please admin merge? Thanks.

**Comment by thiagowfx** [comment]
I'll wait until this is merged before I continue to develop https://github.com/web-platform-tests/wpt/pull/38823.

**Comment by thiagowfx** [comment]
Friendly ping

**Pull request merged**
